api/helpers/sdtd/execute-custom-cmd.js

```diff
@@ -54,11 +54,11 @@ module.exports = {
           );
         }
       } catch (error) {
+        sails.log.error(error);
```

Comment:
these will be super unclear in logs, you should sails.log.error('some prefix', error)